content/en/docs/setup/install/multicluster/single-network/primary/index.md

```diff
@@ -0,0 +1,178 @@
+            <td><code>${CLUSTER_2}</code></td>
+        </tr>
+        <tr>
+            <td><code>network</code></td>
+            <td>Specifies a network ID as an arbitrary string. All clusters in your mesh
+            must be on the same network, and have the same network ID.</td>
+            <td><code>${NETWORK_1}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshID</code></td>
+            <td>Specifies a mesh ID as an arbitrary string. All clusters in your mesh share
+            the same mesh ID.</td>
+            <td><code>${MESH}</code></td>
+        </tr>
+        <tr>
+            <td><code>meshExpansion</code></td>
+            <td>[Recommended] Exposes the Istio control plane through the ingress gateway
+            of the mesh. Enable this option to connect remote clusters to a control
+            plane.</td>
+            <td><code>enabled: true</code></td>
+        </tr>
+    </tbody>
+</table>
+
+Using the previously [set environment variables,](/docs/setup/install/multicluster/#env-var)
+deploy an Istio control plane with the following steps:
+
+1. To pass configuration values to the Istio operator for installation,
+   define a custom resource (CR). Define and save the `install.yaml` CR with
+   the following command:
+
+{{< text bash >}}
+$ cat <<EOF> ${WORK_DIR}/${CLUSTER_2}/install.yaml
+apiVersion: install.istio.io/v1alpha1
+kind: IstioOperator
+spec:
+  values:
+    global:
+      meshID: ${MESH}
+      multiCluster:
+        clusterName: ${CLUSTER_2}
+      network: ${NETWORK_1}
+
+      meshNetworks:
+        ${NETWORK_1}:
+          endpoints:
+          # Always use ‘Kubernetes' as the registry name for
+          # endpoints that reside on the same network as this
+          # instance of the Istio control plane.
+          - fromRegistry: Kubernetes
+          gateways:
+          - registry_service_name: istio-ingressgateway.istio-system.svc.cluster.local
+            port: 443
+      # Expose the control plane through istio-ingressgateway.
+      meshExpansion:
+        enabled: true
+EOF
+{{< /text >}}
+
+1. Install Istio on Cluster-2 with the following command:
+
+    {{< text bash >}}
+    $ istioctl --context=${CTX_2} manifest apply -f \
+    ${WORK_DIR}/${CLUSTER_2}/install.yaml
+    {{< /text >}}
+
+1. Verify that the control plane is ready with the following command:
+
+    {{< text bash >}}
+    $ kubectl --context=${CTX_2} -n istio-system get pod
+    NAME                                    READY   STATUS    RESTARTS   AGE
+    istiod-f756bbfc4-thkmk                  1/1     Running   0          136m
+    prometheus-b54c6f66b-q8hbt              2/2     Running   0          136m
+    {{< /text >}}
+
+1. After the status of all pods is `Running`, you can continue configuring
+   your deployment.
+
+**Congratulations!**
+
+You successfully added a control plane to your primary
+cluster.
+
+Next, configure endpoint discovery to support cross-cluster load balancing.
```

Comment:
Thought: you talk about cross-cluster load balancing a lot.  Has it been defined anywhere?

You may also not want to do load balancing, but instead just have services in cluster A accessible from cluster B.  (This would be common in a regional situation with different deployments)

Reply:
Having services communicate across clusters is what the intent of "cross-cluster LB" is here. Or am I missing something?

We could drop that term altogether and just say:

```
Next, configure cross-cluster endpoint discovery.
```

Reply:
Once the glossary entry for `load balancing` is introduced, a follow up PR can implement mouseovers for it here. An issue to add that entry has been filed already.